src/ecs_cluster/ecs_client.py

```diff
@@ -199,7 +200,7 @@ def get_latest_task_definition_arn(self, cluster_name, service_name, search_tag=
         for task_definition_arn in response['taskDefinitionArns']:
             tags = self.ecs_client.list_tags_for_resource(resourceArn=task_definition_arn).get('tags')
             for tag in tags:
-                if tag['key'] == 'Managed' and tag['value'] == 'ecs-cluster':
+                if tag['key'] == 'Managed' and tag['value'] == search_tag:
```

Comment:
is the expectation for `search_tag` to be always set, if not the error we're returning will have a blank `Managed=`

Reply:
If search tag is unset, `get_latest_task_definition_arn` will just return the most recent task definition